app/src/main/java/com/likeminds/feedsx/deleteentity/model/DeleteEntityExtras.kt

```diff
@@ -0,0 +1,27 @@
+package com.likeminds.feedsx.deleteentity.model
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+class DeleteEntityExtras private constructor(
```

Comment:
why is it `DeleteEntityExtras` it should be `DeleteExtras` only. And change package name as well